applications/openshift/scc/scc_limit_container_allowed_capabilities/rule.yml

```diff
@@ -31,26 +31,16 @@ references:
 ocil_clause: 'allowed capabilities listings in SCCs needs review'
 
 ocil: |-
-    Inspect each SCC returned from running the following command:
-    <pre>$ oc get scc</pre>
-    Next, examine the outputs of the following commands:
-    <pre>$ oc describe roles --all-namespaces</pre>
-    <pre>$ oc describe clusterroles</pre>
-    For any role/clusterrole that reference the
-    <tt>securitycontextconstraints</tt> resource with the <tt>resourceNames</tt>
-    of the SCCs that do not list an explicit <tt>allowedCapabilities</tt>, examine the
-    associated rolebindings to account for the users that are bound to the role.
-    Review each SCC and determine that only required capabilities are either
-    completely added as a list entry under <tt>allowedCapabilities</tt>,
-    or that all the un-required capabilities are dropped for containers and SCCs.
-    varible var_sccs_with_allowed_capabilities_regex can be set to exclude certain
-    SCCs from the check.
-    Use following command to verify if the correct regex is being used, this ouput
-    will list unqualified SCCs:
+    This rule checks the SCCs with allowedCapabilities set to non-null
+    and fails if there are more such SCCs than those allowed in the variable
+    named var-sccs-with-allowed-capabilities-regex. To debug the rule,
+    check the variable value, e.g:
+    <pre>$ oc get variable ocp4-var-sccs-with-allowed-capabilities-regex  -ojsonpath='{.value}' </pre>
+    Then use following command to list the SCCs that would fail the test:
     <pre>$ oc get scc -o json | {{{ jqfilter }}}</pre>
```

Comment:
It may be worth including a sample `jqfilter` as it was included in prior text. Below is an example that gave us the names of SCCs that we needed to add to the regex
```
SCC_REGEX=$(oc get variable ocp4-var-sccs-with-allowed-capabilities-regex -ojsonpath='{.value}' -n openshift-compliance)
oc get scc -o json | jq '[.items[] | select(.metadata.name | test("${SCC_REGEX}") | not) | select(.allowedCapabilities != null) | .metadata.name]'
```